<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertFigura</class>
 <widget class="QDialog" name="insertFigura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>230</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="Punto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Punto</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Punto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="nLati">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Numero lati della figura</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Numero Lati</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Nome della figura</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Annulla">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Annulla inserimento</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string comment="Annulla inserimento" extracomment="Annulla inserimento"> Annulla</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Icone/annulla.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string comment="Annulla inserimento" extracomment="Annulla inserimento"/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Campo info del vertice</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_Nome">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="toolTip">
    <string>Il nome deve iniziare con una lettera. Max 14 caratteri.</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="placeholderText">
    <string>nome-123</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_nLati">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Si possono inserire 3 o 4 lati.</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_X">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Coordinata X</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>Coordinata x</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_Y">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Coordinata Y</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>Coordinata y</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_Info">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Campo info del vertice</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>-24.23</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>165</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Conferma">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Annulla inserimento</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string comment="Annulla inserimento" extracomment="Annulla inserimento">Conferma</string>
   </property>
   <property name="icon">
    <iconset resource="../icone.qrc">
     <normaloff>:/Icone/conferma.png</normaloff>
     <normalon>:/Icone/conferma.png</normalon>
     <activeon>:/Icone/conferma.png</activeon>:/Icone/conferma.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string comment="Annulla inserimento" extracomment="Annulla inserimento"/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Avanti">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Continua inserimento</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <activeon>:/Icone/goOn.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icone.qrc"/>
 </resources>
 <connections/>
</ui>
